generic/tests/fillup_disk.py

```diff
@@ -23,13 +23,21 @@ def run(test, params, env):
     session = vm.wait_for_login(timeout=login_timeout)
     session2 = vm.wait_for_serial_login(timeout=login_timeout)
 
+    mount_lv = params.get("mount_lv", "no")
+    mount_cmd = params.get("mount_cmd")
```

Comment:
I think no need update, mount_cmd is used when mount_lv = yes, and the default value of mount_lv is no.